object_locations:
  - h: [0.04876065, 0.05965745, 0.18542864]
    g: [-62.59071586,  73.86605193, 105.45493455]
    id: "A"
  - h: [-0.03309907,  0.05138066,  0.13824444]
    g: [-14.26008464,  54.74674253,  68.65994041]
    id: "B"
  - h: [-0.04561289,  0.00761808,  0.20859838]
    g: [1.61515498, 59.24698379, 74.22744577]
    id: "C"
  - h: [0.06015607, 0.00690174, 0.30004198]
    g: [29.75074447, 113.98345852, 153.30300393]
    id: "D"
        